acquia/cli
# IDE-134: Support non-production Accounts API

**Motivation**

Allows non-production Cloud API endpoints to be called, which in turn requires using non-production Accounts API.

**Proposed changes**

Add support for `ACLI_CLOUD_API_ACCOUNTS_URI` environment variable to complement `ACLI_CLOUD_API_BASE_URI`. If null, these default to the production endpoints defined in the Acquia PHP SDK. You generally must set both to a matching environment (dev/stage/prod).

**Testing steps**


1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment or [download a pre-built acli.phar](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#automatic-dev-builds) for this PR.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
4. Generate an API key in a non-prod Accounts environment. Install this into `~/.acquia/cloud_api.conf`. Set it as default: `export ACLI_KEY=foo`.
5. `export ACLI_CLOUD_API_ACCOUNTS_URI=foo` and `export ACLI_CLOUD_API_BASE_URI=foo`
6. Run an API command and verify it succeeds: `wacli api:applications:find`
